jobs/search-alert/search_alert/main.py

```diff
@@ -0,0 +1,337 @@
+import click
```

Comment:
A module level docstring would be nice just to provide a high level overview of why this exists and what it does